src/features/autothread.ts

```diff
@@ -39,7 +40,7 @@ const autoThread: ChannelHandlers = {
     }
     // Create threads
     const newThread = await msg.startThread({
-      name: `${msg.author.username} – ${format(new Date(), "HH-mm MMM d")}`,
+      name: createNewThreadName({ username: msg.author.username }),
```

Comment:
Should we make this thread name more generic and recognisable as something that was marked as a poor question? Right now it's just the authors name, maybe we could consider something along the lines of:

`Sorry ${msg.author.username}, your question needs more work`

Reply:
Yeah good idea, it's not specific enough. Your suggestion looks good to me 👍